test/test1009.js

```diff
@@ -0,0 +1,96 @@
+if (typeof exports === 'object') {
+	var assert = require('assert');
+	var alasql = require('..');
+}
+
+describe('Test 1009 - Foreign key error message with table name', function () {
+	const test = '1009';
+
+	before(function () {
+		alasql('create database test' + test);
+		alasql('use test' + test);
+	});
+
+	after(function () {
+		alasql('drop database test' + test);
+	});
+
+	it('A) Single column foreign key - error message should include table and key', function () {
+		alasql('CREATE TABLE tableA (id INT PRIMARY KEY)');
```

Comment:
Please have these multiple calls to alasql as a ` multiple. string with statements seperated by ;